src/main/java/thePackmaster/relics/BagOfHolding.java

```diff
@@ -81,5 +90,49 @@ public String getUpdatedDescription() {
         return DESCRIPTIONS[0] + sb + DESCRIPTIONS[1];
     }
 
+    private static final Map<String, Integer> stats = new HashMap<>();
+    public static String ENERGY_STAT = "energy";
```

Comment:
There's no dependency needed for any of this because the relic stats code looks for these methods by reflection. See https://github.com/ForgottenArbiter/StsRelicStats/blob/184a8584ece3b2feea550d653c16e63976c2c3fd/src/main/java/relicstats/patches/BaseModPatch.java#L28